tools/webhook-ingestion-pipeline/terraform/variables.tf

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### Project Config ###
+variable "project_id" {
+  description = "Project ID for your GCP project"
+  default     = "my-project"
+}
+
+variable "project_num" {
+  description = "Project num for your GCP Project"
+  default     = "my-project-number"
```

Comment:
Don't provide a default for values which *must* be changed.

Reply:
removed these